src/lib/svg2component.js

```diff
@@ -11,23 +11,56 @@ module.exports = (name, svg, isTypeScriptOutput) => {
   const $svg = $('svg');
   toReactAttributes($svg, $);
   const children = $svg.html();
-  const viewBox = $svg.attr('viewBox');
-  const iconJsx = `<Icon viewBox="${viewBox}" {...props}>${children}</Icon>`;
+  const viewBox = $svg.attr('viewBox') ? `viewBox="${$svg.attr('viewBox')}"` : '';
   const code = isTypeScriptOutput ?
     `
       import * as React from 'react';
-      import Icon, {IconProps} from '../Icon';
-      const ${name}: React.SFC<IconProps> = props => (
-        ${iconJsx}
+      export interface ${name}Props extends React.SVGAttributes<SVGElement> {
+        size?: string;
+        width: string;
```

Comment:
width and height are already in SVGAttributes as `number | string`, we don't need them here, just the size

Reply:
make sense